### PR TITLE
add minio guide

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

**Review by MrCreosote — APPROVED** [review]
Can you link to the minio guide from the root readme? Otherwise LGTM

**Comment by MrCreosote** [comment]
maybe the user oriented links should be at the top of the readme come to think of it

**Comment by Tianhao-Gu** [comment]
> maybe the user oriented links should be at the top of the readme come to think of it

👍

File: README.md [hunk 1]
```diff
@@ -128,6 +135,6 @@ Enjoy your Spark journey!
 
 For more information, please consult the [User Guide](docs/user_guide.md).
 
-[
+
```
Comment (MrCreosote) [inline README.md:138]: Can the Accessing the Jupyter Notebook section here be incorporated into the user guide?

Comment (MrCreosote) [reply README.md:138]: oh well, next time maybe. Not a big deal

**Pull request merged**